docs/getting_started/index.md

```diff
@@ -34,8 +34,10 @@ Let's craft a basic "Hello World" scenario:
 ![Hello World Example](hello_world.svg){width=50%}
 
 The graph involves:
-- An input data node named *name*.
-- A task, *build_message*, that processes the *name* data node and outputs a *message* data node.
+
+- An input data node named *input_name*.
+
+- A task, *build_message*, that processes the *input_name* data node and outputs a *message* data node.
```

Comment:
Just wondering about the wording.
Do you want to equate tasks and functions?
(I mean: "A task, referring to the function\*build_message()\*, ...")

Reply:
It is quite the same for me. But I think with your formulation, it also defines what is a task directly